Description: Fix missing DB error during tests
 Tests launched on a clean storage fail because it assumes test DB to exists.
 This patch ensures test DB is created at the beginning of the test run.
Author: Jean Baptiste Favre <debian@jbfavre.org>
Origin: upstream
Last-Update: 2017-09-28
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- /dev/null
+++ b/dbms/tests/queries/0_stateless/99999_prepare.sql
@@ -0,0 +1 @@
+CREATE DATABASE IF NOT EXISTS test;
